Clinical-Genomics/scout
# Export cancer's Coverage and QC report with PDFKit

This PR adds a functionality or fixes a bug:
- Updated the `Coverage and QC` file for the demo cancer sample (note that the export of this exact file in main branch creates a chipped PDF file)
- Fixes #3119 by replacing the lib used to export the report (PDFKit instead of Weasyprint)
- Fixes the missing `Coverage and QC` HTML report for demo cancer case

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Check this case on scout stage (main branch): https://scout-stage.scilifelab.se/cust000/ACC7948A21_ds
2. Install this branch on cg-vm1 and make sure that both HTML and PDF reports work as they should for the given sample

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
